src/libraries/System.Security.Cryptography.Pkcs/tests/Pkcs9AttributeTests.cs

```diff
@@ -20,7 +20,17 @@ public static void Pkcs9AttributeObjectNullaryCtor()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Exception not updated on NetFX")]
         public static void Pkcs9AttributeAsnEncodedDataCtorNullOid()
+        {
+            AsnEncodedData a = new AsnEncodedData(new byte[3]);
+            object ign;
+            Assert.Throws<ArgumentException>("asnEncodedData", () => ign = new Pkcs9AttributeObject(a));
```

Comment:
There's also a helper for this sort of deviation:
```suggestion
            Assert.Throws<ArgumentException, ArgumentNullException>(
                "asnEncodedData",
                "asnEncodedData.Oid",
                 () => ign = new Pkcs9AttributeObject(a));
```